src/mono/netcore/System.Private.CoreLib/src/System/Array.Mono.cs

```diff
@@ -524,14 +522,14 @@ public int GetUpperBound(int dimension)
         [Intrinsic]
         private void GetGenericValueImpl<T>(int pos, out T value)
         {
-            var self = this;
+            Array? self = this;
```

Comment:
Why Array?

Reply:
Looks like an unnecessary annotation added by the fixer.  Opened https://github.com/dotnet/roslyn/issues/42763.